packages/gatsby-source-shopify/README.md

```diff
@@ -396,6 +396,102 @@ The bulk API was chosen for resiliency, but it comes with some limitations, the
 
 <div id="migration-guide"></div>
 
-## Migration guide
+## 🚌 V6 to V7 Migration Guide
 
-We don't currently have a migration guide for v7 of `gatsby-source-shopify`, but it is in the works and will be released shortly after the release of v7.
+Need help upgrading this source plugin from V6 to V7? We want this guide to be as useful as possible. Please open an issue and let us know if you see anything wrong here or find something missing from this guide 🙏
+
+The following breaking schema changes must be updated in your site in order to upgrade:
+
+### `ShopifyProduct` Images/Media
+
+It was previoulsy supported to query for videos or 3D models. In order to add support for these, the `ShopifyProduct` `images` field has been replaced by the `media` field.
```

Comment:
`Previously only querying for images was supported. In order to add support for 3D Models and Videos, the `ShopifyProduct.images` field has been replaced by the `ShopifyProduct.media` union.